.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,27 @@
-## This PR is a:
+## Scope of this PR
 
-- [ ] New topic
-- [ ] Content update
-- [ ] Content fix or rewrite
-- [ ] Bug fix or improvement
+We set labels (and therefore points) based on the scope of changes with a PR.
+Note the scope of this PR by leaving one of the options below and deleting the others.
```

Comment:
Maybe move this line before the labels one to call for action?

Reply:
Done.